### PR TITLE
Audit log fixes

### PR DESCRIPTION
**Pull request by gumbarros**

### CODE REVIEW

File: src/Core/DataManager/Services/FormService.cs
Comment (LucioPelinson) [inline src/Core/DataManager/Services/FormService.cs:1]: Inject class ExpressionsService can be removed

File: src/Core/UI/Components/GridView/JJGridView.cs [hunk 1]
```diff
@@ -669,11 +669,9 @@ protected override async Task<ComponentResult> BuildResultAsync()
     internal async Task<HtmlBuilder> GetHtmlBuilderAsync()
     {
         var html = new HtmlBuilder(HtmlTag.Div);
-        
+        html.WithAttribute("id", Name);
```
Comment (LucioPelinson) [inline src/Core/UI/Components/GridView/JJGridView.cs:672]: Fire..... 0;)

**Review by LucioPelinson — APPROVED** [review]

File: src/Core/UI/Components/GridView/JJGridView.cs (hunk 1 above)
Comment (gumbarros) [reply src/Core/UI/Components/GridView/JJGridView.cs:672]: LGTM :)

**Review by LucioPelinson — CHANGES_REQUESTED** [review]
Review compilation

**Review by LucioPelinson — APPROVED** [review]

**Review by LucioPelinson — APPROVED** [review]

**Pull request merged**